fe/fe-core/src/main/java/org/apache/doris/datasource/property/metastore/HMSProperties.java

```diff
@@ -72,29 +86,50 @@ protected void checkRequiredProperties() {
                         + "but service principal, client principal or client keytab is not set.");
             }
         }
+        if (Strings.isNullOrEmpty(hiveMetastoreUri)) {
+            throw new IllegalArgumentException("Hive metastore uri is required.");
+        }
     }
 
-    public void toPaimonOptionsAndConf(Options options, Configuration conf) {
-        options.set("uri", hiveMetastoreUri);
+    private void checkHiveConfResourcesConfig() {
         Map<String, String> allProps = loadConfigFromFile(getResourceConfigPropName());
-        allProps.forEach(conf::set);
-        conf.set("hive.metastore.authentication.type", hiveMetastoreAuthenticationType);
-        if ("kerberos".equalsIgnoreCase(hiveMetastoreAuthenticationType)) {
-            conf.set("hive.metastore.service.principal", hiveMetastoreServicePrincipal);
-            conf.set("hive.metastore.client.principal", hiveMetastoreClientPrincipal);
-            conf.set("hive.metastore.client.keytab", hiveMetastoreClientKeytab);
+        if (allProps.isEmpty()) {
+            throw new IllegalArgumentException("Hive conf resources config is not empty"
```

Comment:
I think we don't need to check this. let user guarantee this. maybe there is only an empty xml there.